scripts/tmp_remove_files.py

```diff
@@ -0,0 +1,23 @@
+import os
```

Comment:
Why are you removing these files ?

Reply:
This was just a temporary script that I was using to remove previous runs, unnecessary files on margaret, my local machine etc. I plan on removing the script before merging